core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -581,6 +584,10 @@ def _apply_temperature_scaling(self, y_pred_proba):
 
         return y_pred_proba
 
+    def _apply_conformalization(self, y_pred):
+        y_pred += self.conformalize
```

Comment:
maybe add comment to explain this method, code structure looks atypical. Also we need all methods in AbstractModel to be very understandable, as people who want to add custom models to autogluon need to easily understand this class.